CMakeLists.txt

```diff
@@ -240,6 +240,13 @@ if(CMAKE_SYSTEM_PROCESSOR MATCHES "s390x")
   endif(HAS_S390X_MARCH_NATIVE)
 endif(CMAKE_SYSTEM_PROCESSOR MATCHES "s390x")
 
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "loongarch64")
+	CHECK_C_COMPILER_FLAG("-march=native" HAS_LOONGARCH64_MARCH_NATIVE)
+  if(HAS_LOONGARCH64_MARCH_NATIVE)
+    message(STATUS " HAS_LOONGARCH64_MARCH_NATIVE yes")
+  endif(HAS_LOONGARCH64_MARCH_NATIVE)
+endif(CMAKE_SYSTEM_PROCESSOR MATCHES "loongarch64")
```

Comment:
i think you are following the pattern of s390x support. and the whole point of this block is to print `HAS_LOONGARCH64_MARCH_NATIVE yes` if `-march=native` works? as long as it does not have any effect on the compiling process, and only for informative purpose can we just drop it?